.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
   - bundle --version
 bundler_args: --without=development
 
-rvm: 2.0.0
+rvm: 2.2.2
```

Comment:
When the test were ran using 2.0.0 vagrant version 1.8.1 passed but the others failed. Looking through it versions 1.7.2 and 1.6.5 failed because the ruby version wasn't high enough for the version of rack being installed (2.0.3) and for the older versions it wasn't high enough for the version of nokogiri being installed (1.7.2) and i'd assume rack as well if it got to that point.

I'd figured it was pulling the versions selected from the vagrant version so not something that could be controlled from here.